frontend/app/blueprints/sample/templates/sample.html

```diff
@@ -298,102 +304,157 @@ <h4 id="sample-id-heading" class="card-title">{{sample.sample_id}}</h4>
+            <h5>Quality</h5>
+            <hr class="mt-2 col-md-3 border border-success border-2">
+            <div id="qc-heading" class="row gx-5 justify-content-md-center">
+                {% for qc_tool in sample.qc %}
+                    <div class="col-md-auto">
+                        {% if qc_tool.software == "postalignqc" %}
+                            {{ qc_card_postalign(qc_tool) }}
+                        {% else %}
+                            {{ qc_card_generic(qc_tool) }}
                         {% endif %}
-                    {% endfor %}
                     </div>
-                    <h5>Quality</h5>
-                    <hr class="mt-2 col-md-3 border border-success border-2">
-                    <div id="qc-heading" class="row gx-5 justify-content-md-center">
-                        {% for qc_tool in sample.qc %}
-                            <div class="col-md-auto">
-                                {% if qc_tool.software == "postalignqc" %}
-                                    {{ qc_card_postalign(qc_tool) }}
-                                {% else %}
-                                    {{ qc_card_generic(qc_tool) }}
-                                {% endif %}
-                            </div>
-                        {% endfor %}
+                {% endfor %}
 
-                        {% for res in sample.typing_result %}
-                            {% if res.type == "cgmlst" %}
-                                <div class="col-md-auto">
-                                {{ cgmlst_qc_card(res) }}
-                                </div>
-                            {% endif %}
-                        {% endfor %}
-                    </div>
-                    <h5 class="mb-0">Analysis metadata</h5>
-                    <hr class="mt-2 col-md-3 border border-success border-2">
-                    <div class="row">
-                        <div class="col-auto">{{ analysis_meta_pipeline(sample) }}</div>
-                        <div class="col-auto">{{ analysis_meta_databases(sample) }}</div>
+                {% for res in sample.typing_result %}
+                    {% if res.type == "cgmlst" %}
+                        <div class="col-md-auto">
+                        {{ cgmlst_qc_card(res) }}
+                        </div>
+                    {% endif %}
+                {% endfor %}
+            </div>
+            <h5 class="mb-0">Analysis metadata</h5>
+            <hr class="mt-2 col-md-3 border border-success border-2">
+            <div class="row">
+                <div class="col-auto">{{ analysis_meta_pipeline(sample) }}</div>
+                <div class="col-auto">{{ analysis_meta_databases(sample) }}</div>
                 </hr>
             </div>
         </div>
     </div>
 </div>
+<script>
+    const apiURL = "{{ config.BONSAI_API_URL }}"
+    const sampleId = document.getElementById("sample-id-heading").textContent 
+
+    function drawDendrogram(newick, sampleId) {
```

Comment:
Would it make sense to separate out this code into a js-module? Keeping the `sample.html` focused on rendering HTML